data.tf

```diff
@@ -0,0 +1,22 @@
+locals {
+  alarm_create = var.alarm_create && lookup(module.this.tags, "Type", null) != "scheduled" ? 1 : 0
+}
+
+data "gosoline_application_metadata_definition" "main" {
+  count       = local.alarm_create
+  project     = module.this.namespace
+  environment = module.this.environment
+  family      = module.this.namespace
+  application = module.this.name
+  group       = module.this.stage
+}
+
+data "gosoline_application_dashboard_definition" "main" {
+  count       = var.grafana_dashboard_create ? 1 : 0
+  project     = module.this.namespace
+  environment = module.this.environment
+  family      = module.this.namespace
+  application = module.this.name
+  containers  = ["${module.this.stage}-${module.this.name}", "log_router"]
```

Comment:
i'm not sure if we should make assumptions here, but maybe we can keep it for now and make it more flexible later